gradlew

```diff
@@ -28,7 +44,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS='"-Xmx64m"'
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```

Comment:
Just wondering if this is a bit too less memory. Have you checked garbage collector times ?

Reply:
The gradle [docs](https://docs.gradle.org/current/userguide/build_environment.html#sec:configuring_jvm_memory) say:
> The JAVA_OPTS environment variable controls the command line client, which is only used to display console output. It defaults to -Xmx64m